AngularApp/projects/app-service-diagnostics/src/app/shared/components/tools/network-checks/network-check-flows/functionsFlow.js

```diff
@@ -504,6 +648,26 @@ async function validateConnectionStringAsync(propertyName, connectionString, typ
                 title = "Authentication failure";
                 detailsMarkdown = `Authentication failure - the credentials in the configured connection string are either invalid or expired. Please update the app setting with a valid connection string.`;
                 break;
+            case "Managedidentitymissed":
+                title = "Authentication failure";
+                detailsMarkdown = `Managedidentity authentication failure - Credential app setting is not set to managedidentity which disallowed checking connection using clientId.`;

```

Comment:
managed identity is two words.  The message to user here is not clear.  I'll work on rewording separately when we go through the scenarios in a test environment.

Reply:
fixed

Reply:
fixed